ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import { TYPOGRAPHY } from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { GAS_LIMIT } from '../../../../pages/send/send.constants';
+import Button from '../../../ui/button';
+import FormField from '../../../ui/form-field';
+import I18nValue from '../../../ui/i18n-value';
+import Typography from '../../../ui/typography';
+
+import { useAdvanceGasFeePopoverContext } from '../context';
+
+const validateGasLimit = (gasLimit) =>
+  gasLimit < GAS_LIMIT.MIN || gasLimit > GAS_LIMIT.MAX
+    ? 'editGasLimitOutOfBoundsV2'
+    : undefined;
```

Comment:
What do you think about using `null` to mean "no error" instead of `undefined`? I feel like `undefined` should only be reserved to mean "this property doesn't exist" or "this variable doesn't have an assigned value yet".

Reply:
fixed